docs(wallet button): explain new feature wallet button for woocommerce (#1220)
mercadopago/devsite-docs: guides/plugins/woocommerce/introduction.en.md

| Payment cancellations | Cancel the pending payments directly from the platform |
| Binary mode | Approve or reject payments instantly and automatically. |
| Currency conversion | No more compatibility issues. Convert the currency you use in WooCommerce to the currency of your Mercado Pago account. |

> | Payment with card stored in Mercado Pago  | Payment with card stored in Mercado Pago function, for Mercado Pago customers to buy without having to fill in card data at the store's checkout. |

Reviewer: ```suggestion
| Payment with card stored in Mercado Pago  | Allows customers to use their card data saved in Mercado Pago without having to fill in card data at the store's checkout. |
```
